server/src/main/java/io/crate/expression/operator/all/AllOperator.java

```diff
@@ -62,6 +64,15 @@ public abstract sealed class AllOperator<T> extends Operator<T> permits AllEqOpe
         LikeOperators.ALL_NOT_ILIKE
     );
 
+    public static final Set<String> SUPPORTED_COMPARISONS = Set.of(
```

Comment:
Looks like this is only used inside tests, please move it into the test scope then.